tests/unit/test_main.py

```diff
@@ -2073,3 +2073,124 @@ def test_get_flows_by_switch(self, mock_get, _):
         mock_get.return_value.json.return_value = expected
         actual = self.napp.get_flows_by_switch(dpid)
         assert actual == "mocked_flows"
+
+    @patch('napps.kytos.topology.main.Main.get_intf_usage')
+    @patch('napps.kytos.topology.main.Main._delete_interface')
+    async def test_delete_interface_api(self, mock_delete, mock_usage):
+        """Test delete interface API call"""
+        switch_id = "00:00:00:00:00:00:00:01"
+        intf_id = "00:00:00:00:00:00:00:01:1"
+
+        # Error 404 Switch not found
+        endpoint = f"{self.base_endpoint}/interfaces/{intf_id}"
+        response = await self.api_client.delete(endpoint)
+        assert response.status_code == 404, response
+
+        # Error 404 Interface not found
+        mock_switch = get_switch_mock(switch_id)
+        mock_switch.interfaces = {}
+        self.napp.controller.switches = {switch_id: mock_switch}
+        response = await self.api_client.delete(endpoint)
+        assert response.status_code == 404, response
+
+        # Error 409 Interface is used
+        mock_switch.interfaces = {1: Mock()}
+        self.napp.controller.switches = {switch_id: mock_switch}
+        mock_usage.return_value = "There is an allocated TAG."
```

Comment:
You can review this return mocked string here

Reply:
Changed to `It is enabled or active`